### PR TITLE
Use `LengthToFee` in the `congested_chain_simulation`'s fee calculation

### LINKED ISSUE #530: Adjust `congested_chain_simulation` test when we integrate account for the `length` component of the transaction fee
Currently the test disregards the `len` component as it was considered negligible for small txs.
With the change from https://github.com/paritytech/substrate/pull/10785 it might be worthwhile to start consider this component in the calculation of the test.

### PR DESCRIPTION
**Pull request by ghzlatarev**
Signed-off-by: Georgi Zlatarev <georgi.zlatarev@manta.network>

## Description

closes: #530 

* In 0.9.22 update `pallet_transaction_payment` started using `LengthToFee` to scale its fee component that is generated from the transaction byte length.
* Started using this in our `congested_chain_simulation` fee calculation as well.
* No functional changes

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in `<branch>/CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functonality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.
- If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. You can run the `metadata_diff.yml` workflow for help. If this number is updated, then the `spec_version` must also be updated
- Verify benchmarks & weights have been updated for any modified runtime logics

### CODE REVIEW

**Review by Garandor — DISMISSED** [review]
LGTM

File: runtime/manta/src/lib.rs [hunk 1]
```diff
@@ -275,14 +275,14 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    /// Relay Chain `TransactionByteFee` / 10
-    pub const TransactionByteFee: Balance = mMANTA/10;
+    /// Relay Chain `TransactionLengthToFeeCoeff` / 10
+    pub const TransactionLengthToFeeCoeff: Balance = mMANTA/10;
```
Comment (Dengjianping) [inline runtime/manta/src/lib.rs]: `mMANTA/10` => `mMANTA / 10`

**Review by Dengjianping — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,9 +28,9 @@
 - [Dolphin] [\#622](https://github.com/Manta-Network/Manta/pull/622) Update parameter path from `sdk` to `manta-parameters`.
 - [\#636](https://github.com/Manta-Network/Manta/pull/636) Equalize Barrier impl between Calamari/Dolphin production and XCM tests mock parachain.
 - [\#638](https://github.com/Manta-Network/Manta/pull/638) `existence_requirement` argument for `FungibleLedger` trait functions.
-- [\#652](https://github.com/Manta-Network/Manta/pull/652) Reduce CI failure rate by switching AWS CI runners from AMD to Intel.
-- [\#653](https://github.com/Manta-Network/Manta/pull/653) Add concurrency groups for pull request CI builds to reduce CI costs.
-- [\#657](https://github.com/Manta-Network/Manta/pull/657) retire `manta-pc-launch` with `polkadot-launch`.
+- [\#652](https://github.com/Manta-Network/Manta/pull/652) Reduce CI failure rate by switching AWS CI runners from AMD to Intel
+- [\#653](https://github.com/Manta-Network/Manta/pull/653) Add concurrency groups for pull request CI builds to reduce CI costs
+- [\#657](https://github.com/Manta-Network/Manta/pull/657) retire manta-pc-launch with polkadot-launch.
```
Comment (Garandor) [inline CHANGELOG.md]: old changelog is not fixed

**Review by Garandor — CHANGES_REQUESTED** [review]

**Review by Garandor — APPROVED** [review]

**Comment by Garandor** [comment]
relates to OR closes: https://github.com/Manta-Network/Manta/issues/530

pick one

**Comment by ghzlatarev** [comment]
> relates to OR closes: #530
> 
> pick one

Not sure if I'm not getting something, but it always has to be `closes` in order for the PR ot show up linked to the issue in the github project board...

**Comment by Garandor** [comment]
> > relates to OR closes: #530
> > pick one
> 
> Not sure if I'm not getting something, but it always has to be `closes` in order for the PR ot show up linked to the issue in the github project board...

It doesn't have to show up in the project board. it's fine to click into the issue and to the PR from there.
If it `relates to`, the issue should mention the PR in its description ( and it will show open/closed status as well )
That way there's no reopening of issues that have multiple PRs

**Comment by ghzlatarev** [comment]
> > > relates to OR closes: #530
> 
> > > pick one
> 
> > 
> 
> > Not sure if I'm not getting something, but it always has to be `closes` in order for the PR ot show up linked to the issue in the github project board...
> 
> 
> 
> It doesn't have to show up in the project board. it's fine to click into the issue and to the PR from there.
> 
> If it `relates to`, the issue should mention the PR in its description ( and it will show open/closed status as well )
> 
> That way there's no reopening of issues that have multiple PRs

Yea I know you can click on the issue, but when there are many issues it's more convenient to see them on board. That's why we had chosen the other inconvenience of having to reopen issues sometimes over this..

EDIT: actually aren't we trying to solve is the manual work of reopening issues, which we're replacing with the manual work of going back to the issue and editing its description ? Not sure we're gaining anything for the loss of the nice board view with everything plus history.

File: runtime/calamari/src/lib.rs [hunk 3]
```diff
@@ -349,14 +349,14 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    /// Relay Chain `TransactionByteFee` / 10
-    pub const TransactionByteFee: Balance = mKMA / 100;
```
Comment (Dengjianping) [inline runtime/calamari/src/lib.rs:353]: I'm not sure like renaming will keep previous storage in `parameter_types`.

Comment (ghzlatarev) [reply runtime/calamari/src/lib.rs:353]: This is how I create the key for these ---> `let key = hex::encode(Twox128::hash(b":TransactionByteFee:"));`
And when I check the value for this key is always `0x` , also tried with all lower case and upper case as well.

**Review by Garandor — DISMISSED** [review]
Pls add C- and L- labels

**Review by stechu — APPROVED** [review]

**Review by Dengjianping — APPROVED** [review]
lgtm!

**Pull request merged**